addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java

```diff
@@ -340,15 +335,27 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             logger.warn("Serial number missing. Can't send command to device '{}'", getThing());
             return;
         }
-
-        if (channelUID.getId().equals(CHANNEL_SETTEMP) || channelUID.getId().equals(CHANNEL_MODE)) {
-            if (refreshingActuals) {
-                refreshActualsRestore();
-            }
-            SendCommand sendCommand = new SendCommand(maxDeviceSerial, channelUID, command);
-            maxCubeBridge.queueCommand(sendCommand);
-        } else {
-            logger.warn("Setting of channel {} not possible. Read-only", channelUID);
+        switch (channelUID.getId()) {
+            case CHANNEL_SETTEMP:
+                if (refreshingActuals) {
+                    refreshActualsRestore();
+                }
+                Command tempCommand = command;
+                if (command instanceof QuantityType) {
```

Comment:
This breaks the compatibility with OH2.3 see issue #4217
OH2.3 will not send a quanitytype, hence all temp settings are ignored

Reply:
Yes, it does. But why do you want to use this version with OH2.3?

Side note: These lines of code have been moved to
https://github.com/openhab/openhab2-addons/blob/e0ec54d569a387a238c6dee3b2fbf9bfef3a6881/addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java#L419-423

Reply:
if I'm not mistaken, the latest version of the binding is installed.
Hence a OH2.3 (except the release version) will pull the latest one as far as I understand